<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>elevator</class>
 <widget class="QWidget" name="elevator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>380</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_destination">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Buttons</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>201</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Status</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>P</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>门状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_door">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Closed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前楼层</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QSlider" name="verticalSlider_currentFloor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>22</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>61</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_opendoor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开门</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_closedoor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关门</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_alert">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>报警</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_NUM_FLOOR">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_current_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
